app/common/prescription/controllers/patientSimplifiedPrescriptionController.js

```diff
@@ -126,17 +126,17 @@
         vm.prescriptionItem.isArv = false;
         vm.prescriptionItem.interruptedReason = {};
         vm.prescriptionItem.isPlanInterrupted = false;
-        vm.prescriptionItem.therapeuticLine = {};
+        vm.prescriptionItem.arvPlan = {};
```

Comment:
@valvijo, According to our last conversation, it will be necessary to add the therapeutic line of treatment and we will start saving it too in the prescription. Will be done in my next PR